### PR TITLE
enhancement/10.5 - Altitude and speed crossing restrictions

### LINKED ISSUE #10: Command for crossing restrictions
Issuing descent/speed instructions relative to a fix on an aircraft's flightplan is commonplace. For example:

>"AAL482, maintain 190 knots to FINKA"
>"UAL3185, cross LENDY at Flight Level 190 at 250 knots"

Adding a command that would allow for these types of instructions would do a lot for the enroute side of the simulation, whenever that comes along, and will still be very useful in the current version of the sim (especially in very large "super-TRACONs").

Closely related to #11.
Originally reported in zlsa/atc#315.

### Altitude crossing restrictions already completed in #1080.

### PR DESCRIPTION
**Pull request by j-651**
Resolves #10.
Continuation of #1180.

The purpose of this pull request is to introduce speed crossing restrictions and change the command syntax. New syntax: `[CALLSIGN] cross/cr/x [FIX] {a[altitude]} {s[speed]}` (per [here](https://github.com/openscope/openscope/issues/10#issuecomment-583746639)). Example: `AAL123 x aubrn a90 s210`

The order of the speed and altitude entries within the command does not matter.

Much of the groundwork by @JPmAn24

### CODE REVIEW

File: src/assets/scripts/client/aircraft/Pilot/Pilot.js [hunk 1]
```diff
@@ -583,26 +584,73 @@ export default class Pilot {
...
+                return altitudeCheck;
+            }
 
-        const waypoint = this._fms.findWaypoint(fixName);
+            altitude = airportModel.clampWithinAssignableAltitudes(altitude);
 
-        waypoint.setAltitude(altitude);
-        this._mcp.setAltitudeFieldValue(altitude);
-        this._mcp.setAltitudeVnav();
+            const waypoint = this._fms.findWaypoint(fixName);
```
Comment (n8rzz) [inline src/assets/scripts/client/aircraft/Pilot/Pilot.js:597]: If. `fixName` is constant within this method, I'd encourage you to lift this out of and above the conditional blocks so we only need to assign `waypoint` once within this method.

File: src/assets/scripts/client/parsers/aircraftCommandParser/argumentParsers.js [hunk 2]
```diff
@@ -230,8 +230,18 @@ export const timewarpParser = (args = []) => {
...
 export const crossingParser = (args = []) => {
     const fix = args[0];
-    const altitude = convertToThousands(args[1]);
-    // TODO: Add logic for speeds at fix (eg "250K" means to cross at 250kt, while "250" means cross at FL250)
 
-    return [fix, altitude];
+    let altitude;
+    let speed;
+
+    // Set i to 1 to skip fixName
```
Comment (n8rzz) [inline src/assets/scripts/client/parsers/aircraftCommandParser/argumentParsers.js:237]: excellent comment!  thank you!!

File: src/assets/scripts/client/aircraft/Pilot/Pilot.js (hunk 1 above)
Comment (j-651) [reply src/assets/scripts/client/aircraft/Pilot/Pilot.js]: Done 🎉

**Comment by erikquinn** [comment]
Moved to PR #1625 under issue #1624, where it will be merged.

**Pull request merged**